ban/http/schema.py

```diff
@@ -24,6 +24,15 @@
         'url': 'https://adresse.data.gouv.fr/api-gestion/',
     },
     'paths': {},
+    'tags': [
+        {'name': 'Municipality'},
+        {'name': 'Postcode'},
+        {'name': 'Group'},
+        {'name': 'Housenumber'},
+        {'name': 'Position'},
+        {'name': 'Diff'},
+        {'name': 'User'},
+    ],
```

Comment:
I understand class names are uppercased, but maybe we can lowercase it for generating the tags.